rotkehlchen/tests/unit/decoders/test_makerdao.py

```diff
@@ -57,7 +56,118 @@ def test_makerdao_simple_transaction(
             notes='Withdraw 0.6 ETH from ETH-A MakerDAO vault',
             counterparty=CPT_VAULT,
             address=string_to_evm_address('0x809aade1B623d8cDAeb86484d3366A03F8841FBc'),
-            identifier=None,
             extra_data={'vault_type': 'ETH-A'},
         ),
     ]
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('ethereum_accounts', [['0xa217BDa86b0EDb86eE7d4D6e34F493eDF1ea4F29']])  # noqa: E501
+def test_makerdao_withdraw_dai_from_sdai(
```

Comment:
shorter names. No need to prefix with protocol since it's in the filename

```suggestion
def test_withdraw_dai_from_sdai(
```